syncd/SaiDiscovery.cpp

```diff
@@ -289,6 +297,15 @@ std::set<sai_object_id_t> SaiDiscovery::discover(
 {
     SWSS_LOG_ENTER();
 
+    return discover(&startRid, 1);
+}
+
+std::set<sai_object_id_t> SaiDiscovery::discover(
+        _In_ const sai_object_id_t* rids,
+        _In_ size_t count)
```

Comment:
count should be first parameter to keep consistent with all project

Reply:
done